### DISCUSSION TITLE
Add ability to change contrast, gamma, brightness, etc

### DISCUSSION THREAD

**Discussion by R0tten** [RandomEngy/VidCoder#1108] [Ideas]
It would be great to have an ability to set a video filter. Some videos can be too dark or too bright or etc. ffmpeg has a cmd that can do this so not sure if that can be used as a reference?

**Comment by RandomEngy** [comment]
HandBrake has made it clear that their scope is being a video transcoder, not a video editor. So, filters to change how the picture looks would be out-of-scope. Since VidCoder uses the HandBrake engine, it also follows this scope.

You'll need to use a video editing app if you want to make changes to the picture.
